### ISSUE TITLE
set debugger off

### ISSUE THREAD

**Issue by sgrillon14** [SeleniumHQ/selenium#7328]
## 💬 Questions and Help

I want turn off `debugger;` when I use my webdriver

![image](https://user-images.githubusercontent.com/5530550/60095459-e3680880-974e-11e9-8c5f-6843f73d512d.png)

```
ChromeOptions options = new ChromeOptions();
options.addArguments("--disable-  ???? ");     <= ???????

WebDriver driver = new ChromeDriver(options);
driver.get("http://rd.huangpuqu.sh.cn/website/html/shprd/shprd_tpxw/List/list_0.htm");
```

My webdriver is blocked by this and I do not find work arround (by javascript for example)

I try this but not work:

`((JavascriptExecutor) driver).executeScript("debugger; = function(){};");` or `((JavascriptExecutor) driver).executeScript("debugger = function(){};");` or `((JavascriptExecutor) driver).executeScript("window.debugger = function(){};");`

Can you add a new option in this list? : https://peter.sh/experiments/chromium-command-line-switches/

**Comment by lmtierney** [comment]
The issue tracker is not a forum for help. See the template that you deleted before writing this:

### Please note that this issue tracker is not a help form and this issue will be closed.

For questions or help please see:

- [SeleniumHQ IRC channel](https://webchat.freenode.net/)
- [SeleniumHQ Slack channel](https://seleniumhq.herokuapp.com/)
- The [Selenium Users](https://groups.google.com/forum/#!forum/selenium-users) google group

**Comment by sgrillon14** [comment]
@lmtierney, this issue is not a question. An option is missing in selenium and I ask if it is possible to add it because I have not found a workaround.

**Comment by lmtierney** [comment]
> ChromeOptions options = new ChromeOptions();
> options.addArguments("--disable-  ???? ");     <= ???????

All command line arguments for the browser are available via this process. If they are not working, please raise an issue with the chromium/chromedriver team.

**Comment by corevo** [comment]
Also worth noting, `debugger` is a keyword in the language, you can't set it to something else, it would be the same as trying to set `if = 5`.

The only way to disable Chrome's debugger is to not attach one at all (by not opening the devtools), or to click this button 
![image](https://user-images.githubusercontent.com/888886/60103556-7c574d80-9768-11e9-9dcd-a8e7c5f81c8f.png) to disable them, but you'll have to do so manually.

**Comment by sgrillon14** [comment]
@corevo, I want use selenium + Chrome Webdriver (automatic)

**Comment by sgrillon14** [comment]
@lmtierney, I create new issue [here](https://bugs.chromium.org/p/chromedriver/issues/detail?id=2980), 
thank you.
